src/parser.js

```diff
@@ -4,14 +4,16 @@ import findUp from 'find-up'
 import yargs from 'yargs'
 import {oneLine} from 'common-tags'
 import arrify from 'arrify'
-import camelcaseKeys from 'camelcase-keys'
-import chalk from 'chalk'
-import boolify from 'boolify'
 
 const logger = getLogger({prefix: 'prettier-eslint-cli'})
 
 const parser = yargs
-  .usage('Usage: $0 <globs>... [--option-1 option-1-value --option-2]')
+  .usage(
+    `Usage: $0 <globs>... [--option-1 option-1-value --option-2]
+
+Prefix an option with "no-" to set it to false, such as --no-semi to disable
+semicolons and --no-eslint-ignore to disable default ignores.`,
+  )
```

Comment:
I find deindented multi-line strings distracting. Could we use `stripIndent` from [common tags](https://www.npmjs.com/package/common-tags) (already imported)?